pkg/json.go

```diff
@@ -41,26 +41,36 @@ type jsonScorecardResult struct {
 
 //nolint
 type jsonCheckResultV2 struct {
-	Details []string
-	Score   int
-	Reason  string
-	Name    string
+	Details []string `json:"details"`
```

Comment:
The `json` annotations are not needed. Remove?

Reply:
I've renamed all JSON fields to be lower-case. Alternatively, I could try renaming the go structure's fields. wdut?

Reply:
IIUC, the Go struct fields need to be exposed to json encoder so cannot be lower case. Why do we want the JSON output to be lower case though?

Reply:
everywhere I look (OSV, GitHub API, etc), the fields are lower-case.

Reply:
I see. I'm ok with changing to lower case. These changes will affect the cron job (v2 version) though. So I would prefer holding off on this PR until we have decoupled the two.

Reply:
is there anything in particular that you think may break with this PR for v2? The BQ table needs to be updated, what else?

Reply:
BQ schema change for sure, can't think of anything else.